### DISCUSSION TITLE
how to make completion docs appear on top of completion?

### DISCUSSION THREAD

**Discussion by aldzikridp** [autozimu/LanguageClient-neovim#1175] [General]
I'm using jdtls, and the docs appear under completion
![2020-12-16-192849_Screenshot](https://user-images.githubusercontent.com/34727793/102349181-8660b200-3fd5-11eb-8c91-36e442b5983d.png)

**Comment by martskins** [comment]
I think what you are seeing is a consequence of lack of screen space. It's not clear to me if you want the docs to appear on top of it or above it though. The docs normally appear next to the completion menu, but it gets a little tricky when there's no space to show both, so things like that may happen. It should be possible to include some logic to find the best space to show the docs though, if someone is willing to give it a try I think it could be a nice first contribution.
